docs/refs.bib

```diff
@@ -46,6 +46,22 @@ @article{Bandyopadhyay_2015_Limitations
   month=jun, pages={3593-3604} 
 }
 
+@article{Bennett_1992_Communication,
+  title = {Communication via one- and two-particle operators on Einstein-Podolsky-Rosen states},
```

Comment:
```suggestion
  title = {Communication via one- and two-particle operators on {E}instein-{P}odolsky-{R}osen states},
```

(otherwise, these names won't render as capital letters).